component/form/form.css

```diff
@@ -0,0 +1,55 @@
+.input-field {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0.6rem 0.5rem;
+  border-radius: 0.3rem;
+  border: 1px solid var(--light-grey);
+  margin-bottom: 0.5rem;
+}
+#email,
```

Comment:
Use classes instead of I'd if May need this input fields in different places, then it will be an issue.

Reply:
I am using right now but later when I will push to dev branch then I'll remove it !!